controllers/appointment.controller.js

```diff
@@ -0,0 +1,120 @@
+const HealthTrackRecord = require("../models/healthTrackRecord.model");
+const constants = require("../utils/constants");
+const User = require("../models/user.model");
+const Appointmet = require('../models/appointment.model');
+
+exports.addAppointment = async (req, res) => {
```

Comment:
User should pass hospital id, doctor id.. 

So we need to first very if the hospital id and doctor id is valid
and doctor is working in that hospital